www/md_base/src/app/common/directives/panel/panel.directive.coffee

```diff
@@ -0,0 +1,30 @@
+class Panel extends Directive
+
+    constructor: ->
+        return {
+            restrict: 'E'
+            templateUrl: 'views/panel.html'
+            controller: '_PanelController'
+            controllerAs: 'panel'
+            transclude: true
+            bindToController: true
+            scope:
+                title: '@'
+                isCollapsed: '='
+        }
+
+
+class _Panel extends Controller
+
+    constructor: (@$element, @$scope) ->
+        @$scope.$watch 'panel.isCollapsed', (=> @updateCollapse())
```

Comment:
no need for parens: 

    @$scope.$watch 'panel.isCollapsed', => @updateCollapse()